bionic/cache.py

```diff
@@ -899,7 +899,7 @@ class Provenance(object):
     @classmethod
     def from_computation(
             cls, code_fingerprint, case_key, dep_provenance_digests_by_task_key,
-            treat_bytecode_as_functional):
+            treat_bytecode_as_functional, is_functionally_volatile):
```

Comment:
Do we still want to use the term "volatile"?

Reply:
hmm, I wanted to let the method know that we should update the functional part and adding changes per run to it made the variable name too long.

Reply:
Now that I think of it, `can_functionally_change_per_run` is not that long really.